conda-recipe/meta.yaml

```
@@ -53,6 +53,7 @@ outputs:
         - numpydoc >=1.0.0
         - pandas >=1.3.3
         - partsegcore-compiled-backend >=0.15.11
+        - bermuda >=0.1.4
```

```suggestion
        - bermuda >=0.1.5
```

https://github.com/conda-forge/bermuda-feedstock/pull/4